internal/providers/docker/components/container/build.go

```diff
@@ -22,31 +25,37 @@ import (
 )
 
 func (c *Container) Build(ctx context.Context, input *api.BuildInput) (*api.BuildOutput, error) {
-	if c.canBuild() {
-		if err := c.buildImage(ctx); err != nil {
+	if c.State.Image.Spec == "" {
+		// SEE NOTE: [MIGRATE_CONTAINER_STATE].
+		return nil, errors.New("refresh needed")
```

Comment:
When will this refresh happen? Do people need to run the manual `exo refresh` command?